scripts/data_cleaning.py

```diff
@@ -1,7 +1,9 @@
-import os
 import click
```

Comment:
We might want to add a brief script description: 
```suggestion
'''
Clean the dataset and by recording and removing missing rows and features with many missing values.
'''
import click
```